Platforms/QemuQ35Pkg/QemuFlashFvbServicesRuntimeDxe/FwBlockService.h

```diff
@@ -194,4 +194,9 @@ SetPcdFlashNvStorageBaseAddresses (
   VOID
   );
 
+VOID
+UpdateOvmfFlashVariablesEnable (

```

Comment:
Any particular reason why you have Ovmf in the name of this new function?  Given that we've gotten rid of the package.

Reply:
Not anymore~ will update.

Reply:
Renamed to QEMU flash variable.